packages/marketplace/src/components/ui/app-detail-modal/app-detail-inner.tsx

```diff
@@ -95,13 +106,16 @@ export const AppDetailInner: React.FunctionComponent<AppDetailInnerProps> = ({
   setStateViewUninstall,
   installationsSetFormState,
   loginMode,
+  isAdmin,
+  isCurrentLoggedUserClient,
 }) => {
   if (appDetailModalState === 'VIEW_DETAIL_BROWSE') {
+    const canInstall = !(!isAdmin && isCurrentLoggedUserClient)
```

Comment:
Suggest to change `isAbleToInstall = isAdmin && !isCurrentLoggedUserClient`

Reply:
should be `isAdmin || !isCurrentLoggedUserClient`

Reply:
hi @tanphamhaiduong @vuhuucuong
I changed it to ` const isInstallBtnDisabled = isCurrentLoggedUserClient && !isAdmin`
because the Install button only disabled when user logged as CLIENT and not have
admin permission.